### PR TITLE
Add stat for clock runtime

### PR DESCRIPTION
**Pull request by djKooks**
Following https://github.com/fastly/Viceroy/issues/76
(please ignore branch/commit name...)

### CODE REVIEW

File: lib/src/execute.rs [hunk 1]
```diff
@@ -228,7 +229,7 @@ impl ExecuteCtx {
         remote: IpAddr,
     ) -> Result<(), ExecutionError> {
         info!("handling request {} {}", req.method(), req.uri());
-
+        let now = Instant::now();
```
Comment (cratelyn) [inline lib/src/execute.rs:232]: how about renaming this variable ever so slightly, to something like...

```suggestion
        let start_timestamp = Instant::now();
```

File: lib/src/execute.rs [hunk 2]
```diff
@@ -276,11 +277,15 @@ impl ExecuteCtx {
             .expect("`memory` is exported")
             .size(&store);
 
+        let distant = Instant::now().duration_since(now);
```
Comment (cratelyn) [inline lib/src/execute.rs:280]: ...and similarly, a slight name tweak

```suggestion
        let request_duration = Instant::now().duration_since(start_timestamp);
```

File: lib/src/execute.rs [hunk 3]
```diff
@@ -276,11 +277,15 @@ impl ExecuteCtx {
...
+
         info!(
             "request completed using {} of WebAssembly heap",
             bytesize::ByteSize::kib(heap_pages as u64 * 64)
         );
 
+        info!("request completed in {:?}", distant);
```
Comment (cratelyn) [inline lib/src/execute.rs:287]: same rename as above, just applying it here so you can fix this with the click of a mouse :computer_mouse: 

```suggestion
        info!("request completed in {:?}", request_duration);
```

**Review by cratelyn — COMMENTED** [review]
nice work @djKooks! i have just a _small_ pair of nitpicks about variable names. otherwise, this looks great :slightly_smiling_face:

**Comment by djKooks** [comment]
@cratelyn updated 🙏

**Comment by cratelyn** [comment]
this looks like it's almost ready to go @djKooks! there's currently a hiccup in CI here that was addressed in #123. if you rebase this branch on the current `main` we should be able to fix that. (_note: happy to help walk you through that if you run into any questions!_)

**Comment by djKooks** [comment]
@cratelyn got it. Updated~

**Pull request merged**